language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/field_access_ctx_config1.json

```diff
@@ -212,11 +212,11 @@
     {
       "label": "iterator()",
       "kind": "Function",
-      "detail": "object {public isolated function next() returns record {| map:Type value; |}? ;}",
+      "detail": "map:$anonType$_3",
```

Comment:
Here, the type descriptor's signature is identified as an anon_type (which is the internal representation). We should be able to get the previous one. `object {public isolated function next() returns record {| map:Type value; |}? ;}`

Reply:
Ack. In that case, should it be `object {public isolated function next() returns record {| map:Type value; |}? ;}` or `map:object {public isolated function next() returns record {| map:Type value; |}? ;}`

Reply:
We shouldn't show the qualified name reference unless it is a type definition